### DISCUSSION TITLE
media from less and less sites can now be downloaded.

### DISCUSSION THREAD

**Discussion by onyxmxyno** [aclap-dev/video-downloadhelper#1283] [Help and Q&A]
### Your question

at first, with the new app for firefox I realized that downloading videos from eronity.com is not possibly anymore (I thought a security measure from their end)
but now, after updating firefox today to the most recent version (130.0.1), not also downloading mp3 from sueddeutsche.de did stop working as well. 
now I think both are not due tu security measured at the website but something at your end.

### Addon & CoApp version and details

firefox: 130.0.1
vdh: 9.1.0.44
coApp: 2.0.19.0

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Post some URLs of specific problem web pages.  Don't say anything on the sites will do.  I'm not familiar with the sites.  Make it easy for me to help you.  Post URLs of pages on which problem content appears.  I'll try to download the items myself & see what happens.

**Comment by knallie** [comment]
Downloads on bigmarker.com are not working. It seems to work but the download file content is trash.

**Reply by GrampaWildWilly** [reply]
I say, "Don't say anything on the sites will do."  So what do you do?

**Reply by knallie** [reply]
I don't understand what you mean.

**Reply by GrampaWildWilly** [reply]
URLs.  We need URLs of pages on which there is content you are having trouble downloading.  The idea is to provide enough information to allow somebody else to investigate the problem, to see if they can experience the same problem you do.  That is not possible on a site that requires a user ID & password, but we have alternative things we can try in that case.  But step 0 is to provide URLs of actual web pages where there is problematic content.  Not just the name of a web site.  Specific URLs.

**Reply by knallie** [reply]
How can I pm you here?
Usually websites block URLs in messages, github doesn't?

**Reply by GrampaWildWilly** [reply]
Look around this forum.  Read other discussions.  We are full of URLs here.  PMing is not possible.

**Reply by GrampaWildWilly** [reply]
This is mostly an international self-help cooperative of users.  Users.  The idea is to post things in public.  There's 2 main reasons for this.  First is you get to rely on a larger population to help you.  More people can read what you say & any given problem might get multiple folks trying to help.

Second is the discussion persists so others can read it.  It doesn't help somebody a week or 6 months from now coming here for advice if a chunk of the advice happened out of sight in back channels.

So PM goes against the fundamental philosophy of a forum like this.  This concept applies to most public discussion forums similar to this one.  Even if PM were possible, I would avoid it.

**Comment by Mattmjs** [comment]
@knallie , bigmarker uses content protection #1472
